tests/helics/application_api/FilterAdditionalTests.cpp

```diff
@@ -54,16 +54,23 @@ TEST_P(filter_type_tests, message_reroute_filter_object1)
 
     mFed->requestTimeAsync(1.0);
     fFed->requestTime(1.0);
+    // std::this_thread::sleep_for(std::chrono::milliseconds(2000));
+    // auto res = broker->query("root", "global_time_debugging");
+    // std::this_thread::sleep_for(std::chrono::milliseconds(2000));
+    // std::this_thread::yield();
+    // auto res2 = broker->query("root", "global_time_debugging");
```

Comment:
Maybe something else to remove when not needed, along with a few other similar comments in other test files.